qiskit/providers/fake_provider/backends/perth/fake_perth.py

```diff
@@ -0,0 +1,29 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+
+"""
+Fake Perth device (7 qubit).
+"""
+
+import os
+from qiskit.providers.fake_provider import fake_backend
+
+
+class FakePerthV2(fake_backend.FakeBackendV2):
```

Comment:
If there is no V1 variant we don't need to mark this as V2

```suggestion
class FakePerth(fake_backend.FakeBackendV2):
```

Reply:
sounds good. So, the type is the way to check the version?

Reply:
You can either type check, or there is a `.version` class attribute set in the base abstract class. So you can do something like: `backend.version >= 2` or something like that.